events/_event-post-macros.html

```diff
@@ -131,19 +168,24 @@
 
    Create HTML event meta date/time markup when given:
 
-   date: A post date from a query result.
+   date:                        A post date from a query result.
+
+   datetime_format (optional):  An object to use for customizing event-meta time and
+                                date formats.
 
    ========================================================================== #}
 
-{% macro event_meta_datetime(date) %}
+{% macro event_meta_datetime(date, datetime_format={}) %}
+    {%- set date_format =  datetime_format.date_format | default('%b %m, %Y', true)   -%}
+    {%- set time_format =  datetime_format.time_format | default('%H:%M %p %Z', true) -%} 
+
     <time class="event-meta_datetime dt-start" datetime="{{ date | date('%Y-%m-%d %H:%M %p %Z') }}">
```

Comment:
Uppercase Z at the end doesn't seem right, that'll give the [name for the timezone](https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior), so that format would kick out e.g. `2015-03-02T01:15:20EST`. I didn't read through all of http://microformats.org/wiki/value-class-pattern but it looks like it's asking for the timezone offset +/-0000, which would be lowercase z.